targets/TARGET_STM/TARGET_STM32F7/TARGET_STM32F756xG/device/TOOLCHAIN_IAR/stm32f756xg.icf

```diff
@@ -26,7 +26,7 @@ ITCM-RAM:  16K (0x3FFF)   @0x00000000
 if (!isdefinedsymbol(MBED_ROM_START)) { define symbol MBED_ROM_START  = 0x8000000; }
 if (!isdefinedsymbol(MBED_ROM_SIZE)) { define symbol MBED_ROM_SIZE   = 0x200000; }
 if (!isdefinedsymbol(MBED_RAM_START)) { define symbol MBED_RAM_START  = 0x20010000; }
-if (!isdefinedsymbol(MBED_RAM_START)) { define symbol MBED_RAM_SIZE   = 0x40000; }
+if (!isdefinedsymbol(MBED_RAM_SIZE)) { define symbol MBED_RAM_SIZE   = 0x40000; }
```

Comment:
+1

But as I am a copy paste expert... you need to update also stm32f746xg.icf, stm32f767xi.icf, and stm32f769xi.icf...

Reply:
how come they don't fail the build? will check

Reply:
Fixed now

Reply:
Thanks a copy paste expert!